core/field_dropdown.js

```diff
@@ -122,6 +122,7 @@ Blockly.FieldDropdown.prototype.init = function() {
   });
   this.arrow_.setAttributeNS('http://www.w3.org/1999/xlink',
       'xlink:href', Blockly.mainWorkspace.options.pathToMedia + 'dropdown-arrow.svg');
```

Comment:
This looks good.  Please make this change upstream as well.

Reply:
I do not think this applies to blockly, which seems to use the same class name on dropdown fields as other fields.